src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const listArrays = sourceString
+    .split(';')
+    .filter((item) => item.length)
+    .map((element) => element.split(':').map((item) => item.trim()));
+
+  const resultObject = {};
```

Comment:
The variable name `resultObject` is too generic according to the checklist item 2. Consider using a name that reflects its purpose, such as `styleObject`.

Reply:
ok